src/api.js

```diff
@@ -0,0 +1,19 @@
+export const BASE_URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api';
+
+export const GAME_ID = 'sa7iLyWCcC3Ao7tJMfAb';
+
+export const getScores = () => fetch(`${BASE_URL}/games/${GAME_ID}/scores`)
+  .then((resp) => resp.json());
+
+export const addScore = (username, score) => fetch(
+  `${BASE_URL}/games/${GAME_ID}/scores/`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      user: username,
+      score,
+    }),
+  },
+).then((resp) => resp.json());
```

Comment:
- [ ] Good job done here! However, for this project, you are required to use `async` and `await` with the `fetch` method to send and receive data. Let's fix that

Reply:
Thank you I'll make these improvements.